app/controllers/prototypes_controller.rb

```diff
@@ -17,14 +17,14 @@ def new
 
   def create
     @prototype = current_user.prototypes.new(prototype_params)
-    @prototype.save ? (redirect_to root_path) : (render :new)
+    @prototype.save ? (redirect_to root_path, notice: 'prototype is scussesfully created!!')  : (render :new, alert: 'wops somthing went wrong!')
```

Comment:
Line is too long. [145/80]<br>Unnecessary spacing detected.<br>Operator `:` should be surrounded with a single space.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.